### PR TITLE
Jra config primer edits

### PR DESCRIPTION
**Pull request by jamsden**

### CODE REVIEW

File: notes/link-guidance.html [hunk 1]
```diff
@@ -3,32 +3,36 @@
...
 class="remove"
-></script>
+></script> -->
+<script src="http://127.0.0.1:9000/respec-oasis-common.js" async class="remove"></script>
+
 <script class='remove'>
   var respecConfig = {
-      specStatus:           "PND", //PN Draft
+      shortName:            "link-guidance",
+      specStatus:           "PN", //PN Draft
```
Comment (berezovskyi) [inline notes/link-guidance.html:16]: #558 makes changes to the same file. What's the difference?

**Comment by berezovskyi** [comment]
Jim, seems like you have updated all notes, does this PR combine all the changes from #545? Shall we close #545 once we merge this PR?

**Comment by jamsden** [comment]
I believe so.

File: notes/link-guidance.html (hunk 1 above)
Comment (jamsden) [reply notes/link-guidance.html:16]: None, just an editing that happened in the wrong branch. Its hard to use different branches when I need to see similar changes I made on another branch that hasn't been merged to master yet.

**Review by berezovskyi — APPROVED** [review]

**Pull request merged**